### PR TITLE
Admin Login with ActiveAdmin

### PR DESCRIPTION
**Pull request by flomosq**
# Admin Login with ActiveAdmin

### Trello Card #30

- Change project for activeadmin
- ActiveAdmin and webpacker installation
- Fix tests
- Fix linting issues
- Admin user model tests

### CODE REVIEW

File: app/admin/admin_users.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  form do |f|
+    f.inputs do
+      f.input :email
+      f.input :password
+      f.input :password_confirmation
+    end
+    f.actions
+  end
+end
```
Comment (afmicc) [inline app/admin/admin_users.rb:37]: what about show view? I think if you don't configure it, you would be shown the passwords 😬

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```
Comment (afmicc) [inline db/seeds.rb]: leave an empty line below

**Review by afmicc — APPROVED** [review]
Great job! 👏

File: app/admin/dashboard.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    #       end
+    #     end
+    #   end
+
+    #   column do
+    #     panel "Info" do
+    #       para "Welcome to ActiveAdmin."
+    #     end
+    #   end
+    # end
```
Comment (brunvez) [inline app/admin/dashboard.rb]:
```suggestion
```

File: app/admin/admin_users.rb (hunk 1 above)
Comment (flomosq) [reply app/admin/admin_users.rb:37]: I checked and the passwords aren't shown

File: db/migrate/20210304202615_create_active_admin_comments.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[6.1]
+  def self.up
+    create_table :active_admin_comments do |t|
+      t.string :namespace
+      t.text   :body
```
Comment (brunvez) [inline db/migrate/20210304202615_create_active_admin_comments.rb]: 👀

**Review by brunvez — APPROVED** [review]

**Pull request merged**